src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -320,6 +334,18 @@ void OnPlatformEventHandler(const chip::DeviceLayer::ChipDeviceEvent * event, in
     }
 }
 
+// Get the attestation challenge for the current session in progress. Only valid when called
+// synchronously from inside a CommandHandler.
+// TODO: Create an alternative way to retrieve the Attestation Challenge without this huge amount of calls.
+ByteSpan GetAttestationChallengeFromCurrentSession(app::CommandHandler * commandObj)
+{
+    VerifyOrDie((commandObj != nullptr) && (commandObj->GetExchangeContext() != nullptr));
+
+    ByteSpan attestationChallenge =
+        commandObj->GetExchangeContext()->GetSessionHandle()->AsSecureSession()->GetCryptoContext().GetAttestationChallenge();
```

Comment:
So this is safe because this is only used for commands that need Admin privilege, so we know the session is CASE or PASE because you can't get admin privileges for group?

Might be worth documenting that assumption in terms of who calls this function.

Reply:
Done